packages/@aws-cdk/aws-pipes-targets-alpha/lib/eventBridge.ts

```diff
@@ -0,0 +1,145 @@
+import { IInputTransformation, IPipe, ITarget, TargetConfig } from '@aws-cdk/aws-pipes-alpha';
+import { IEventBus } from 'aws-cdk-lib/aws-events';
+import { IRole } from 'aws-cdk-lib/aws-iam';
+
+/**
+ * EventBridge target properties.
+ */
+export interface EventBridgeTargetParameters {
+  /**
+   * The input transformation to apply to the message before sending it to the target.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-pipetargetparameters.html#cfn-pipes-pipe-pipetargetparameters-inputtemplate
+   * @default none
+   */
+  readonly inputTransformation?: IInputTransformation;
+
+  /**
+   * A free-form string, with a maximum of 128 characters, used to decide what fields to expect in the event detail.
+   *
+   * Minimum: 1
+   * Maximum: 128
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-pipetargeteventbridgeeventbusparameters.html#cfn-pipes-pipe-pipetargeteventbridgeeventbusparameters-detailtype
+   * @default none
+   */
+  readonly detailType?: string;
+
+  /**
+   * The URL subdomain of the endpoint.
+   *
+   * For example, if the URL for the endpoint is https://abcde.veo.endpoints.event.amazonaws.com, then `EndpointId` is abcde.veo.
+   *
+   * Pattern: `^[A-Za-z0-9\-]+[\.][A-Za-z0-9\-]+$`
+   * Minimum: 1
+   * Maximum: 50
```

Comment:
I would use an `@example` tag here, something like
```suggestion
   * @example
   * // if the URL for the endpoint is https://abcde.veo.endpoints.event.amazonaws.com
   * 'abcde.veo'
```